File: Latihan3/Latihan3_CPP/Ram.cpp
#include <string>

using namespace std;

class Ram
{
private:
	/* data */
	// class attribute
	string capacity;
	int price;

public:
	Ram(/* args */){};

	Ram(string capacity, int price)
	{
		this->capacity = capacity;
		this->price = price;
	};

	// getter and setter
	void setCapacity(string capacity)
	{
		this->capacity = capacity;
	};

	string getCapacity()
	{
		return this->capacity;
	};

	void setPrice(int price)
	{
		this->price = price;
	};

	int getPrice()
	{
		return this->price;
	};

	// destructors
	~Ram(){};
};

File: Latihan3/Latihan3_PHP/Ram.php
<?php

class Ram
{
	// class attribute
	private $capacity;
	private $price;

	// constructors
	public function Ram($capacity = "", $price = 0)
	{
		$this->capacity = $capacity;
		$this->price = $price;
	}

	// getter and setter
	public function setCapacity($capacity)
	{
		$this->capacity = $capacity;
	}

	public function getCapacity()
	{
		return $this->capacity;
	}

	public function setPrice($price)
	{
		$this->price = $price;
	}

	public function getPrice()
	{
		return $this->price;
	}

	// destructors
	function __destruct()
	{
	}
}
